svc/api/cloud/src/route/games/mod.rs

```diff
@@ -614,3 +616,28 @@ pub async fn complete_banner_upload(
 
 	Ok(json!({}))
 }
+
+fn gen_name_id(s: impl AsRef<str>) -> String {
+	let proc_ident = util::format::str_to_ident(s);
+
+	// Default
+	let proc_ident = if proc_ident.is_empty() {
+		"game"
```

Comment:
this has a high chance of collision, since we're only taking 3 samples of the alphanum. not a huge deal, but maybe we should sample 8 characters if we use the default "game" name